dd-library-php-setup.php

```diff
@@ -280,7 +280,12 @@ function install_appsec($options, $selectedBinaries)
         print_error_and_exit(
             "The version of the AppSec package provided/downloaded is " .
             $tarball_version .
-            " but the minimum version supported by this installer is 0.2.0\n");
+            " but the minimum version supported by this installer is 0.3.0\n");
+    }
+
+    $rulesPathSetting="rules_path";
+    if (version_compare($tarball_version, "0.3.0", ">=")) {
+        $rulesPathSetting="rules";
```

Comment:
If the minimum version supported is 0.3.0, does it make sense to have support for the old ini setting name? Also, you might want to upgrade the old name to the new name, in case someone set the old name already (though that's unlikely).